Services/DataBaseService/Interfaces/IRepository.cs

```diff
@@ -1,7 +1,10 @@
-﻿namespace DataBaseService.Interfaces
+﻿using System;
+
+namespace DataBaseService.Interfaces
 {
     public interface IRepository<Input>
     {
         void Create(Input data);
+        void Delete(Guid data);
```

Comment:
How about
   ```
 public interface IRepository<Input, Id>
    {
        void Create(Input data);
        void Delete(Id data);
```
or something? Don't know if it's the right way, but the approach seems more flexible

Reply:
If you fix sth, you have to write "done" or "fix"
Not just resolve conversation